x-pack/plugins/security_solution/server/endpoint/routes/resolver/entity/utils/build_resolver_entity.ts

```diff
@@ -9,6 +9,8 @@ import type * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
 import { getFieldAsString, supportedSchemas } from './supported_schemas';
 import type { ResolverEntityIndex } from '../../../../../../common/endpoint/types';
 
+const toArray = <T>(input: T | T[]) => ([] as T[]).concat(input);
```

Comment:
Can the `lodash` concat method be used here instead?

Reply:
How would you suggest the code look like?

Reply:
Just asking if there is an existing utility we can use instead if creating our own

Reply:
Not sure, I copied this one from another place in Kibana, not sure if it's worthy to export it. Haven't seen anything similar around ;/